src/components/FeedbackPrompt.js

```diff
@@ -82,9 +79,16 @@ class FeedbackPrompt extends React.Component {
     });
 
     return (
-      <Modal open={this.props.isOpen}>
+      <Modal
+        size='large'
+        open={this.props.isOpen}
+        onClose={this.close}
+        closeOnDimmerClick={false}
+        closeOnEscape={false}
```

Comment:
I love closing on escape, but maybe that's just me ;)
Also, not something we have to mess with right now.

Reply:
Fair! I disabled it though because I wanted to make it harder to accidentally close the modal and delete your answers.

I also think showing a 2nd warning modal on close with some nonempty answers could be a good idea.